src/cephadm/cephadm

```diff
@@ -5168,7 +5168,7 @@ def prepare_ssh(
         cli(['orch', 'apply', 'crash'])
 
     if not ctx.skip_monitoring_stack:
-        for t in ['prometheus', 'grafana', 'node-exporter', 'alertmanager']:
+        for t in ['prometheus', 'grafana', 'node-exporter', 'alertmanager', 'promtail', 'loki']:
```

Comment:
Discussed this a bit with others, and thinking it would be best to include these new daemons behind a new flag. Something like

```
if ctx.centralized_logging:
  for t in ['promtail', 'loki']:
    logger.info('Deploying %s service with default placement...' % t)
    cli(['orch', 'apply', t])
```
after adding a new `centralized_logging` flag to bootstrap with a small addition around here https://github.com/ceph/ceph/blob/main/src/cephadm/cephadm#L9178-L9242. We just don't want too many services attached to a single flag.

Reply:
Agree to add new flags.
In order to have same manner as other flags, It would be better to name the flag  'skip_centralized_logging'. However, this will deploy promtail, loki as default.
What do you think about this?

Reply:
I think that's alright. I'd probably avoid backporting that change however, as I think deploying new daemon types by default is something better suited for a major release. If that's okay with you I'm fine with a `skip_centralized_logging` flag